content/_releases/22/26.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Product Update - v22.26
+description: Details of the product update v22.26 from 30th of June 2022.
+layout: rel
+version: "22.26"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-06-30
+redirect_from:
+  - /releases/22.26.html
+---
+
+## Improvements and Updates
+
+### Node.js Sailor version `2.6.28`
+
+We released a new version (`2.6.28`) of the Node.js Sailor library version. This
+version includes an additional mechanism to handle errors from maester during the
+lightweight message upload.
+
+### Edit Snapshot button always present
+
+Now it is possible to edit/set the snapshot for any step of any draft even if the
+flow has not been published or executed.
+
+{% include img.html max-width="80%" url="/assets/img/RN/2226/RN2226-snapshot.png" title="Edit Snapshot always available" %}
+
+
+{% if site.data.tenant.name == "elastic.io" %}
+
+## OEM Related
+
+Information in this section is intended for our customers who use
+[OEM version of the elastic.io platform](https://www.elastic.io/saas-embedded-integration/).
+
+### Email domain blacklist (continuation)
+
+This is further improvement of email domain blacklist feature from [22.22](/releases/22/22#email-domain-blacklist)
+release. Now you can use wildcard character (`*`) with a domain name to exclude all variants
+like `gmail.*`, `yahoo.*`, etc.
+
+
+{% endif %}
+
+## Fixed bugs
+
+*   Now you can not edit data samples in read only mode.
```

Comment:
Context is needed here, otherwise this sentence is senseless - Read only mode - WHAT is it & WHY is it useful. HOW do I activate it.

Here is how it is changed to allow you to...